pkg/controller/provider/validation.go

```diff
@@ -360,6 +360,21 @@ func (r *Reconciler) inventoryCreated(provider *api.Provider) error {
 	return nil
 }
 
+func (r *Reconciler) handleServerCreationFailure(provider *api.Provider, err error) {
+	provider.Status.Phase = ConnectionAuthFailed
+	msg := fmt.Sprint("The OVA provider server creation failed:", err)
```

Comment:
need to add space at the end of the first string:
```suggestion
	msg := fmt.Sprint("The OVA provider server creation failed: ", err)
```